torch/functional.py

```diff
@@ -821,6 +822,22 @@ def btriunpack(LU_data, LU_pivots, unpack_data=True, unpack_pivots=True):
                      unpack_data=unpack_data, unpack_pivots=unpack_pivots)
 
 
+def btrisolve(b, LU_data, LU_pivots, out=None):
+    r"""Solves the system of equations :math:`Ax = b` using the partially pivoted LU
+    factorization of :math:`A` given by :attr:`LU_data` and :attr:`LU_pivots`.
+
+    For more information regarding :func:`torch.btrisolve`, please check
+    :func:`torch.lu_solve`.
+
+    .. warning::
+        :func:`torch.btrisolve` is deprecated in favour of :func:`torch.lu_solve` and will be
+        removed in the next release. Please use :func:`torch.lu_solve` instead.
+    """
+    warnings.warn("torch.btrisolve is deprecated in favour of torch.lu_solve and will be "
+                  "removed in the next release. Please use torch.lu_solve instead.", stacklevel=2)
+    return torch.lu_solve(b=b, LU_data=LU_data, LU_pivots=LU_pivots, out=out)
```

Comment:
@vishwakftw I think `b=b` is wrong.  Calling this directly gets the following:

```
TypeError: lu_solve() missing 3 required positional argument: "input", "LU_data", "LU_pivots"
  File "/usr/local/fbcode/gcc-5-glibc-2.23/lib/python3.6/unittest/case.py", line 59, in testPartExecutor
```

Reply:
(this should probably just be `torch.lu_solve(b, LU_data=LU_data, LU_pivots=LU_pivots, out=out)`). I don't know how native_functions.yaml names the first argument.

Reply:
I've changed it. Sorry about this.

Reply:
No worries, I missed these on my initial review